cmd/ci-secret-mirroring/README.md

```diff
@@ -0,0 +1,31 @@
+# CI-Secret-Mirroring
+
+This tool extends the [openshift/ci-secret-mirroring-controller](https://github.com/openshift/ci-secret-mirroring-controller)
+to support mirroring secrets cross Kubernetes/OpenShift-clusters.
+
+We use `--kubeconfig` to specify the path to a [kube config](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+that the tool will load and use it to access clusters for reading and writing secrets.
+
+It expects a configuration like the one below which specifies the mapping from the source
+of the mirroring and the targets.
+
+```yaml
+secrets:
+- from:
+    cluster: default
```

Comment:
We could fix `default` for now as discussed.

How are the secrets generated in `default`? We can still use our BW-script.
An advantage (wrt implementation) of fix source in default: we listen to events only from ONE cluster.

Reply:
@stevekuznetsov Do you want this tool to read from BW as well?